cobigen/cobigen-core-api/src/main/java/com/devonfw/cobigen/api/util/MavenUtil.java

```diff
@@ -340,4 +342,26 @@ public static Path getProjectRoot(Path inputFile, boolean topLevel) {
     LOG.debug("Project root could not be found.");
     return null;
   }
+
+  /**
+   * Gets a list of download URLs by groupId from the specified repository search REST API
+   *
+   * @param repositoryType String of the type of the repository e.g. maven, jfrog, nexus
+   * @param groupId the groupId to search for
+   *
+   * @return List of artifact URLS
+   */
+  public static List<URL> getMavenArtifactsByGroupId(String repositoryType, String groupId) {
+
+    List<URL> artifactList = new ArrayList<>();
```

Comment:
Is this line necessary? Why cannot be directly assigend to the method call?

Reply:
Removed it now.